templates/upstart.conf.j2

```diff
@@ -7,4 +7,4 @@ respawn
 setuid {{ gunicorn_user }}
 chdir {{ gunicorn_app_dir }}
 
-exec gunicorn --config /etc/gunicorn/{{ gunicorn_app_name }}.py {{ gunicorn_wsgi }}
+exec gunicorn --timeout {{ gunicorn_timeout }} --config /etc/gunicorn/{{ gunicorn_app_name }}.py {{ gunicorn_wsgi }}
```

Comment:
Is there a reason why this setting is being applied via command-line argument vs. configuration file?